src/irc/core/irc-servers.c

```diff
@@ -61,6 +62,7 @@ void irc_servers_reconnect_init(void);
 void irc_servers_reconnect_deinit(void);
 
 static int cmd_tag;
+static char *rejoin_channels_mode;
```

Comment:
Any particular reason this is in this scope and not inside irc_server_get_channels?